### PR TITLE
add api call on plugin activation

### PR DESCRIPTION
**Pull request by nicolasThal**

### CODE REVIEW

**Comment by nicolasThal** [comment]
[(2) Etq Uwordpress, les stats adback remontent dès que j'ai activé le plugin](https://trello.com/c/s6p86UYM/665-2-etq-uwordpress-les-stats-adback-remontent-des-que-j-ai-active-le-plugin)

File: adback-solution-to-adblock/class-ad-back.php [hunk 1]
```diff
@@ -27,7 +27,7 @@ public function getContents($url)
         }
     }
 
-    public function postContents($url, $fields, $header = array())
+    public static function postContents($url, $fields, $header = array())
```
Comment (gignonje) [inline adback-solution-to-adblock/class-ad-back.php]: You have a call in `saveMessage()` to `$this->postContents(...)`, you should update it to `self::postContents()` I think

Comment (julienguilmont) [inline adback-solution-to-adblock/class-ad-back.php]: Or, you can instantiate a new Ad_Back_Generic object (this object have no dependencies) in activator :)

File: adback-solution-to-adblock/includes/class-ad-back-activator.php [hunk 2]
```diff
@@ -37,6 +37,21 @@ public static function activate()
 
         $charset_collate = $wpdb->get_charset_collate();
 
+        $fields = [
+            'email' => get_bloginfo('admin_email'),
+            'website' => get_site_url(),
+        ];
+        $response = Ad_Back_Generic::postContents('https://adback.co/tokenoauth/register/en', $fields);
```
Comment (julienguilmont) [inline adback-solution-to-adblock/includes/class-ad-back-activator.php]: I think the treat of the request (and this response) must be in Ad_Back_Generic.

**Comment by nicolasThal** [comment]
@gignonje @julienguilmont pr updated

**Pull request merged**